docs/MainSite/Installation.md

```diff
@@ -14,7 +14,7 @@ See LICENSE.txt for details.
 * [Charm++](http://charm.cs.illinois.edu/) 6.8 or newer (must be compiled from source)
 * [Git](https://git-scm.com/)
 * BLAS (e.g. [OpenBLAS](http://www.openblas.net))
-* [Blaze](https://bitbucket.org/blaze-lib/blaze/overview)
+* [Blaze](https://bitbucket.org/blaze-lib/blaze/overview) v3.2
```

Comment:
Is it worth enforcing this in CMake?

Reply:
Yes, I'll have to see if there's a way to do that. I posted a similar question as an issue on the Blaze repo: https://bitbucket.org/blaze-lib/blaze/issues/138/check-blaze-version